### ISSUE TITLE
A11y_ML .NET Model Builder_ Scenario_ Keyboard: User is not able to navigate to the previous control using "Shift + tab" when keyboard focus is on any of the tab items 

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1146]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev3; #NonBenchmark; #Visual Studio; #Keyboard; #WCAG2.4.3

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 preview 6.0

### Repro Steps:

1.Open Visual Studio Enterprise 2019 Preview 
2. Create a new .Net Core Console App (C#) 
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Scenario" tab item and now use "Shift + tab" to navigate back to the previous control and verify if its working

### Actual Result:
User is not able to navigate to the previous control using "Shift + tab" when keyboard focus is on any of the tab items 

### Expected Result:
User should be able to navigate back to previous control using "Shift + tab"

### User Impact:
Keyboard only users will get confused if user is not able to navigate to the previous control using "Shift + tab"

### Attachment for Reference:

[1146_ML .NET_User is not able to navigate to the previous control using Shift + tab when keyboard focus is on any of the tab items.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5656771/1146_ML.NET_User.is.not.able.to.navigate.to.the.previous.control.using.Shift.%2B.tab.when.keyboard.focus.is.on.any.of.the.tab.items.pptx)

**Comment by SaisreeDandamudi** [comment]
Verified the bug in the latest build and the issue is fixed. Hence closing the bug
Please find the attachment for reference
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0
[1146_ML.NET_fixedUser.is.not.able.to.navigate.to.the.previous.control.using.Shift.+.tab.when.keyboard.focus.is.on.any.of.the.tab.items.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/6007854/1146_ML.NET_fixedUser.is.not.able.to.navigate.to.the.previous.control.using.Shift.%2B.tab.when.keyboard.focus.is.on.any.of.the.tab.items.pptx)
